ghost/admin/app/styles/layouts/content.css

```
@@ -1019,6 +1019,27 @@
+    color: var(--midgrey);
+    font-weight: 500;
+    text-transform: capitalize;
+}
+
+.gh-content-entry-nav svg {
+    width: 11px;
+    height: 11px;
+}
+
+.gh-content-entry-nav svg path {
+    stroke: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)